decidim-core/lib/decidim/content_block_manifest.rb

```diff
@@ -63,9 +46,13 @@ def settings(&block)
     private
 
     def image_names_are_unique
-      errors.add(:image_names, :invalid) if image_names.count != image_names.uniq.count
+      image_names = images.map { |image| image[:name] }
+      errors.add(:images, "names must be unique per manifest") if image_names.count != image_names.uniq.count
```

Comment:
I18n?

Reply:
Not needed, this will be executed on application startup, the error is not for end users.